message.ts

```diff
@@ -478,6 +520,222 @@ export type MessageEntity =
   | MessageEntity.TextLinkMessageEntity
   | MessageEntity.TextMentionMessageEntity;
 
+/** This object contains information about the quoted part of a message that is replied to by the given message. */
+export interface TextQuote {
+  /** Text of the quoted part of a message that is replied to by the given message */
+  text: string;
+  /** Special entities that appear in the quote. Currently, only bold, italic, underline, strikethrough, spoiler, and custom_emoji entities are kept in quotes. */
+  entities: MessageEntity[];
```

Comment:
```suggestion
  entities?: MessageEntity[];
```
`entities` is also marked as optional: https://core.telegram.org/bots/api#textquote